RiotSharp.AspNetCore/RiotSharp.AspNetCore.csproj

```diff
@@ -1,7 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <PackageId>RiotSharp.AspNetCore</PackageId>
+    <Version>1.0.0</Version>
+    <PackageVersion>1.0.0</PackageVersion>
```

Comment:
How can this be the first major release since it's the first release? Let's start with 0.1.0 and when we're confident we've got something working we'll make it 1.0.0.